hosting/templates/hosting/orders.html

```diff
@@ -3,95 +3,65 @@
-                                    <div class="modal-content">
-                                        <div class="modal-header">
-                                            <button type="button" class="close" data-dismiss="modal"
-                                                    aria-label="Confirm"><span
-                                                    aria-hidden="true">&times;</span>
-                                            </button>
-                                        </div>
-                                        <div class="modal-body">
-                                            <div class="modal-icon"><i class="fa fa-trash" aria-hidden="true"></i></div>
-                                            <h4 class="modal-title" id="ModalLabel">{% trans "Do you want to delete your order?"%}</h4>
-
-                                            <form method="post"
-                                                  action="{% url 'hosting:delete_order' order.id %}">
-                                                {% csrf_token %}
-                                                <div class="modal-footer">
-                                                    <button type="submit" class="btn btn-danger">{% trans "Delete"%}
-                                                    </button>
-                                                </div>
-                                            </form>
-                                        </div>
-                                    </div>
-                                </div>
-                            </div>
-                        {% endfor %}
-                        </tbody>
-                    </table>
-
-                    {% if is_paginated %}
-                        <div class="pagination">
-			            <span class="page-links">
-			                {% if page_obj.has_previous %}
-                                <a href="{{ request.path }}?page={{ page_obj.previous_page_number }}">{% trans "previous"%}</a>
-                            {% endif %}
-                            <span class="page-current">
-			                    Page {{ page_obj.number }} of {{ page_obj.paginator.num_pages }}.
-			                </span>
-                            {% if page_obj.has_next %}
-                                <a href="{{ request.path }}?page={{ page_obj.next_page_number }}">{% trans "next"%}</a>
-                            {% endif %}
-			            </span>
-                        </div>
+    <table class="table table-switch">
+        <thead>
+            <tr>
+                <th>{% trans "Order Nr." %}</th>
+                <th>{% trans "Date" %}</th>
+                <th>{% trans "Amount" %}</th>
+                <th>{% trans "Status" %}</th>
+                <th></th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td class="xs-td-inline" data-header="{% trans 'Order Nr.' %}">{{ order.id }}</td>
```

Comment:
Nr indeed seems to be the shorthand for number in German https://en.wikipedia.org/wiki/Numero_sign
I would never use it in English, just saying ..